src/randoop/main/Help.java

```diff
@@ -5,13 +5,13 @@
 import java.util.Arrays;
 import java.util.List;
 
+import randoop.Globals;
```

Comment:
This reordering is weird to me.  It seems that before, the utility classes were first (JDK classes, then plume-lib).  Now what is the order?